CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v36.0.0...main)
 
+* Android
+  * BUGFIX: `TimespanMetricType.measure` and `TimingDistributionMetricType.measure` won't get inlined anymore ([#1560](https://github.com/mozilla/glean/pull/1560)).
+    This avoids a potential bug where a `return` used inside the closure would end up not measuring the time.
+    Use `return@measure <val>` for early returns.
```

Comment:
Should this make it to the docs?

Reply:
It probably should, but right now we're not even calling out `measure` at all^^

Reply:
Ok, would you kindly file a follow up for this?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1699653